source/bson-corpus/tests/symbol.json

```diff
@@ -50,31 +50,31 @@
     "decodeErrors": [
         {
             "description": "bad symbol length: 0 (but no 0x00 either)",
-            "bson": "0C0000000261000000000000"
+            "bson": "0C0000000E61000000000000"
```

Comment:
Note: the only change here was swapping `02` for `0E`. Similar to code w/o scope, symbol and string have the same structure.